README.md

```diff
@@ -53,7 +53,9 @@ Most of the components that make up the platform can even be completely embedded
 
 ## Contributing
 
-Please see our [contribution guidelines](CONTRIBUTING.md) for how to raise issues and how to contribute code to our project.
+Camunda 7 Community Edition will reach its end of life by next October and the Enterprise Edition will move into maintenance mode, receiving only critical updates from that point forward. Looking ahead, [Camunda 8](https://github.com/camunda/camunda) is the successor to this project, so we strongly encourage you to explore it and contribute there instead.
```

Comment:
🔧 Can we rephrase this slightly and move this further up to the top of the README file?

```markdown
> [!Caution]  
> Camunda 7 Community Edition will reach its **end of life by October 2025** and the Enterprise Edition will move into maintenance mode, receiving only maintenance improvements as well as bug and security fixes from that point forward. Looking ahead, [Camunda 8](https://github.com/camunda/camunda) is the successor to this project, so we strongly encourage you to explore it and contribute there instead.
```

![image](https://github.com/user-attachments/assets/329910a6-0dd8-489e-b929-ab67416d5091)

Reply:
Sure thing. Any opinion on rephrasing [the automatic comment](https://github.com/camunda/team-automation-platform/pull/265#issuecomment-2541564786)?